delta/sdk/src/main/resources/contexts/acls.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "nxv": "https://bluebrain.github.io/nexus/vocabulary/",
+    "_path": "nxv:path",
+    "AccessControlList": "nxv:AccessControlList",
```

Comment:
It can just be
```json
{
  "@context": {
      "nxv": "https://bluebrain.github.io/nexus/vocabulary/",
      "@vocab": "https://bluebrain.github.io/nexus/vocabulary/",
      "_path": "nxv:path"
   },
  "@id": "https://bluebrain.github.io/nexus/contexts/acls.json"
}
```